test/unit/aws/conftest.py

```diff
@@ -0,0 +1,5 @@
+from test.aws.conftest import (
```

Comment:
this is a bad idea, don't import from conftest, if you want to externalize fixtures use https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file

Reply:
I will replace the import by plugins in next push.